components/41-organisms/image-collections/image-collections.bindings.js

```diff
@@ -20,3 +20,8 @@
   });
 
 })();
+
+
+module.exports = {
+  hidden: true
+};
```

Comment:
Shouldn't this be place in the `image-collections.config.js`? 🤨

Reply:
Yes! I misread!